src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -284,8 +310,23 @@ const Ownership = () => {
               Component={<Input type={'email'} {...register('email')} />}
               label={t('organizers.ownerships.request_modal.email')}
               error={
-                formState.errors.email &&
-                t(`organizers.ownerships.request_modal.email_error`)
+                formState.errors.email && (
+                  <Trans
+                    i18nKey={`organizers.ownerships.request_modal.${
+                      isNonExistingUserError
+                        ? 'email_not_known_error'
+                        : 'email_error'
+                    }`}
+                    components={[
+                      <Link
+                        key="link"
+                        href={`https://profile.uitid.be/${i18n.language}`}
+                      >
+                        Will be replaced
```

Comment:
Just to double check, "Will be replaced" is a placeholder not the copy right?

Reply:
Yes the `Will be replaced` will get replaced by the content from the translation file